cmd/convert.go

```diff
@@ -113,9 +113,9 @@ func newConvertCmd() *cobra.Command {
 			}
 
 			if !force {
-				err = conf.Validate()
-				if err != nil {
-					return err
+				verrs, err := conf.Validate()
```

Comment:
maybe some comments about verrs vs err

Reply:
I documented them in Validate()'s docstring.